packages/g/google-cloud-sdk/package.yml

```diff
@@ -15,6 +15,8 @@ description: |
     A set of command-line tools for the Google Cloud Platform. Includes gcloud (with beta and alpha commands), gsutil, and bq.
 builddeps  :
     - pkgconfig(python3)
+patterns   :
+    - docs : [/usr/share/man]
```

Comment:
For consistency's sake, can this be moved under the `install` section and be in the form of:

```yaml
patterns   :
    - docs :
        - /usr/share/man
```

Reply:
Done